lib/utils/isStandardSyntaxAtRule.cjs

```diff
@@ -9,6 +9,11 @@
  * @returns {boolean} If `true`, the declaration is standard
  */
 function isStandardSyntaxAtRule(atRule) {
+	// Ignore `@charset` css rule (is parsed as at-rule)
+	if (atRule.name.toLowerCase() === 'charset') {
+		return false;
+	}
```

Comment:
fix
medium

> [@charset must be written literally, i.e., the 10 characters `@charset "` (lowercase, no backslash escapes), followed by the encoding name, followed by `";`.](https://drafts.csswg.org/css2/#charset)

i.e. you shouldn't lowercase `atRule.name`, it wouldn't be valid anyway

Reply:
Either `@charser` was written correctly and then it isn't a real `at-rule`, or it was written incorrectly, also making it not a standard `at-rule`.

In both cases it is never a standard syntax `at-rule` and could be ignored by all these rules :)

I don't think we will ever want to process `@CHARSET "utf-8";` but not one of:
- `@charset "utf-8"`
- `@charset  "utf-8"` (two spaces)
- `@charset 'utf-8'` (single quote)

Either we want to ignore any `@charset` and all its invalid variations or we want to target `@charset` specifically to make sure it was written correctly. (i.e. an `at-charset-rule-no-invalid` rule)